packages/components/src/public/files/FileAttachmentForm.tsx

```diff
@@ -41,20 +41,22 @@ interface FileAttachmentFormProps {
     // map between extension and SizeLimitProps.  Use "all" as the key for limits that apply to all formats.
     // "all" limits will be overridden by limits for a specific extension.
     sizeLimits?: Map<string, FileSizeLimitProps>;
-    sizeLimitsHelpText?: React.ReactNode;
+    sizeLimitsHelpText?: ReactNode;
     showAcceptedFormats?: boolean;
     allowDirectories?: boolean;
+    includeDirectoryFiles?: boolean;
```

Comment:
Careful when linting here as these inline comments can get thrown out of order. I typically hand edit these out of order property listings.